src/handlerService.js

```diff
@@ -1,16 +1,52 @@
-export function getEventHandler(event, stripes, module, data) {
-  const m = module.getModule();
-  const eventHander = m[module.handlerName];
-  if (!eventHander) return null;
+/**
+ * handleEvent
+ * Call the module's event handler and return the result. A handler that wants
+ * to intercede will return a component to be rendered. A handler that ignores
+ * the event or merely invokes side-effects will return null.
+ *
+ * @param {string} event from ./events.js
+ * @param {object} stripes
+ * @param {object} handler a component with an event-handling function
+ * @param {object } data shaped like { displayName, name, module} values from
+ *   the module the event was invoked on
+ * @returns Component to intercede, null otherwise
+ */
+export function handleEvent(event, stripes, handler, data) {
```

Comment:
The function `getEventHandler` is public https://github.com/folio-org/stripes-core/blob/main/index.js#L67. The name should not be changed for other modules.